test/In.ProjectEKA.HipServiceTest/Discovery/FilterTest.cs

```diff
@@ -13,88 +12,156 @@ namespace In.ProjectEKA.HipServiceTest.Discovery
     [Collection("Patient Filter Tests")]
     public class FilterTest
     {
+        private class Linda

```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6377136)